### PR TITLE
Initial workflow settings

### PR DESCRIPTION
**Pull request by egil**

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -24,7 +29,7 @@
     <AnalysisMode>AllEnabledByDefault</AnalysisMode>
 		<EnableNETAnalyzers>true</EnableNETAnalyzers>
 		<AnalysisLevel>latest</AnalysisLevel>
-		<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+		<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
```
Comment (perkops) [inline Directory.Build.props]:
```suggestion
		<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
```

File: src/Directory.Build.props [hunk 2]
```diff
@@ -4,24 +4,69 @@
   <!--
     Add any shared properties you want for the projects under this directory that need to be set before the auto imported Directory.Build.props
   -->
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\directory.build.props" />
```
Comment (perkops) [inline src/Directory.Build.props]: Shouldnt this be Directory.Build.props - casing is very important for msbuild on *NIX

File: test/Directory.Build.props [hunk 3]
```diff
@@ -4,7 +4,7 @@
   <!--
     Add any shared properties you want for the projects under this directory that need to be set before the auto imported Directory.Build.props
   -->
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\directory.build.props" />
```
Comment (perkops) [inline test/Directory.Build.props]: Shouldnt this be Directory.Build.props - casing is very important for msbuild on *NIX

File: src/Directory.Build.props (hunk 2 above)
Comment (egil) [reply src/Directory.Build.props]: It is indeed important. I just thought we were going lowercase on these, but can see the file in question is actually still in title case.

**Review by perkops — APPROVED** [review]

**Review by davidkallesen — APPROVED** [review]

**Pull request merged**